Dockerfile

```diff
@@ -14,4 +14,4 @@ RUN chown -R www-data:www-data $DATAPUSHER_DIR
 
 ENV LC_ALL=C
 ENTRYPOINT ["./datapusher-entrypoint.sh"]
-CMD /usr/local/bin/uwsgi --ini-paste $DATAPUSHER_DIR/deployment/datapusher-uwsgi.ini
+CMD /usr/local/bin/uwsgi --ini-paste $DATAPUSHER_DIR/deployment/datapusher-uwsgi.ini --honour-stdin
```

Comment:
`--honour-stdin` should be configured in the ini file.
I think it might be needed only for development (to make pdb working), if that's the case are you sure it's safe to have it turn on in prod? Maybe it could be turned on only in development (via DEBUG env or some other way)?